Requests/mandataris-export.ts

```diff
@@ -0,0 +1,45 @@
+import { Request } from 'express';
+
+import { HttpError } from '../util/http-error';
+
+export function mandatarisExportRequest(request: Request) {
```

Comment:
:ant: feels a bit odd to do this in a separate file shouldn't this just be in the route? it's not something to be reused and this way it's an extra hop between files to see what's going on.  (ant is for 'muggenziften')

Reply:
Hm gonna keep it as a separate file. The logic in the router should be readable as you said and not include a big mapping imo. Separate file for one piece of specific logic.

> still thinking about this..